internal/mev/blxr/health.go

```diff
@@ -0,0 +1,46 @@
+package blxr
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"sync"
+	"time"
+
+	log "github.com/sirupsen/logrus"
+)
+
+func (c *Client) KeepAlive(ctx context.Context, locker sync.Locker) error {
+	locker.Lock()
+	defer locker.Unlock()
```

Comment:
It looks like this doesn't make calls to Paloma.  That's why we need the lock elsewhere.  Is the lock needed here?

Reply:
Ah, I thought it was to limit network activity or something. I will remove it, this part is completely independent from Paloma.